server/homebrew.api.js

```diff
@@ -282,13 +287,21 @@ const deleteBrew = async (req, res)=>{
 					throw { status: 500, message: 'Error while removing' };
 				});
 		} else {
+			if(googleId && isOwner) {
```

Comment:
Add a comment here too explaining this case of moving the Google brew to HB so other authors retain access.